### ISSUE TITLE
[Easy] Implement POL_Download in bash

### ISSUE THREAD

**Issue by qparis** [PlayOnLinux/POL-POM-5#7]
Currently, POL_Download is implemented in Python: 
This feature need to be implemented in bash for backward compatibility. 
As the feature is already implemented in the java core, the bash implementation will be straightforward.

The following code will download a script.

```python
from com.playonlinux.framework import SetupWizard
from com.playonlinux.framework import Downloader

setupWindow = SetupWizard("Test script")
Downloader(setupWindow).get("http://files.metaquotes.net/metaquotes.software.corp/mt4/mt4setup.exe").check("e42ebc9ebc2d8975de5a0c94386d1a6a")

setupWindow.close()
```

To achieve that you need to: 
 * Complete bash/lib/scripts (following the same structure than the commands you can find in bash/lib/setupWindow)
```bash
POL_Download ()
{
	# Download a file and place it in the current directory
	# Usage: POL_Download [URL] [MD5]

        # To be implemented
}
```

 * Add a parser case in SetupWindowCommandParser.py responding to the command you have created before.

**Comment by 4thguy** [comment]
I want to see if I understood your request correctly.

You want a script that performs a download (wget), checks the file against a hash, and returns a success or fail. Then you want to check the result using Python in SetupWindowCommandParser.py.

**Comment by qparis** [comment]
Not exactly, the download part has already been implemented.

Another guy has already started on this one, have a look at the pull request list :-)

**Comment by 4thguy** [comment]
OK, thanks for the heads up.

My Python-fu is very weak, but hopefully I'll be able to contribute more with the Java aspects of the project.

**Comment by qparis** [comment]
Thanks :-)

You can have a look at the pull request and do the same for POL_SetupWindow_free_presentation if you want

**Comment by qparis** [comment]
Fixed by commit 632b240aa6f09d4b47f2d2b022a530d975600e59
Many thanks to seijikun
